docs/metrics/cohen-kappa.md

```diff
@@ -0,0 +1,114 @@
+# Cohen's Kappa
+
+Cohen's Kappa is a statistical metric that evaluates the reliability of an algorithm's predictions with similar goals
+as [accuracy](./accuracy.md) or [F<sub>1</sub>-score](./f1-score.md). However, the Cohen Kappa score magnifies the
+advantage a classifier has over a random classifier based on class frequencies. The Kappa statistic is a robust metric,
+particularly when there is significant class imbalance for binary and multiclass classification problems, which
+extends to more complicated ML tasks such as object detection.
+
+## Implementation Details
+
+Cohen's Kappa coefficient (\(\kappa\)) is calculated by comparing the observed agreement
+([true positives and true negatives](./tp-fp-fn-tn.md)) between the model inferences and the ground truths against
+expected classifications by random chance based on the marginal frequencies of each class.
+
+The formula for Cohen's Kappa is:
+
+$$
+\kappa = \frac{P_o - P_e}{1 - P_e}
+$$
+
+where:
+
+- \(P_o\) is the `observed agreement`
+- \(P_e\) is the hypothetical probability of `chance agreement`
+
+??? question "Understanding \(P_o\) and \(P_e\)"
+    [Accuracy](./accuracy.md) measures the overall correctness of predictions. Cohen's Kappa adjusts for any agreement
+    that may happen by chance, providing a more detailed understanding of the classifier's performance.
+
+    **\(P_o\) (Observed Agreement)**:
+    The proportion of times where the model and the ground truth agree. This is the number of instances
+    where the predicted labels match the true labels, divided by the total number of instances
+    (i.e. [accuracy](./accuracy.md) of the model).
+
+
+    **\(P_e\) (Chance Agreement)**: We can simplify the chance agreement calculation into three steps.
+
+    1. **Calculate class proportions**:
+    For each class \(c\), calculate the proportion of instances predicted as class \(c\) by the model, denoted
+    as \(p_{c,\text{pred}}\). Do the same for instances among ground truths, denoted as \(p_{c,\text{true}}\).
+
+    2. **Compute chance agreement for each class**:
+    For each class \(c\), calculate the chance agreement by multiplying the model's proportion for class \(c\) with
+    the ground truth's proportion for class \(c\):
+
+        $$
+        \text{chance}_{c} = p_{c,\text{pred}} \times p_{c,\text{true}}
+        $$
+
+    3. **Sum chance agreements across all classes**:
+    Sum the calculated chance agreements for all classes to get the total probability of chance agreement. This is
+    the overall probability that the model and the ground truths would agree on the classification of instances by
+    chance alone, across all classes:
+
+        $$
+        P_e = \sum_{c} \text{chance}_{c}
+        $$
+
+There is an alternative method using [TP, FP, FN, and TN](./tp-fp-fn-tn.md):
```

Comment:
How would this alternative method work for multiclass?

Reply:
It would be very similar to F1-Scores - borrowing sections from [docs/f1-score](https://docs.kolena.com/metrics/f1-score/)

Reply:
I think this would be an useful thing to add to the doc — what do you think?